net/pfSense-pkg-ntopng/files/usr/local/pkg/ntopng.inc

```diff
@@ -155,7 +155,7 @@ function ntopng_sync_package() {
 	/* Create rc script */
 	$stop = <<<EOD
 # Kill ntopng and redis
-	/usr/bin/killall ntopng redis-cli redis-server
+	while [ "$(pgrep ntopng)" != "" ]; do /usr/bin/killall ntopng && sleep 3; done; /usr/bin/killall redis-cli redis-server
```

Comment:
Please use pgrep's -q parameter and also split it in multiple lines to improve readability:
```
while pgrep -q ntopng; do
    /usr/bin/killall ntopng
    sleep 3
done
/usr/bin/killall redis-cli redis-server
```

Reply:
Ok, np 😄

Reply:
Checked on my own instance before commit*